src/averageAges.js

```diff
@@ -14,12 +16,22 @@
  *
  * @return {number}
  */
-function calculateMenAverageAge(people, century) {
+function calculateMenAverageAge(peopleMen, century) {
```

Comment:
we got all people here, why peopleMen, function return average age of men but it accept all people